sklearn/kernel_approximation.py

```diff
@@ -752,6 +752,9 @@ def fit(self, X, y=None):
         ----------
         X : array-like of shape (n_samples, n_features)
             Training data.
+
+        landmarks: array-like of shape (n_components,) or list
+             indices of the samples that will be included in the landmark set.
```

Comment:
Need a check that landmarks is longer or equal to n_components

Given that landmarks might be longer than n_components, `inds = landmarks`, vs `basis_inds = landmarks[: n_components]`

Reply:
Done

Reply:
We now take care of the case where the number of landmarks is less than the number of the number of components, but if the number of landmarks is smaller, we should probably issue a warning to note that only the provided landmarks will be selected, in the same spirit as the warning for `n_components > n_samples` in L767.